packages/ember-data/lib/system/model/errors.js

```diff
@@ -91,19 +91,8 @@ var MapWithDefault = Ember.MapWithDefault;
   @uses Ember.Enumerable
   @uses Ember.Evented
  */
-export default Ember.ArrayProxy.extend(Ember.Evented, {
-  /**
-    Register with target handler
+export default Ember.ArrayProxy.extend({
 
-    @method registerHandlers
-    @param {Object} target
-    @param {Function} becameInvalid
-    @param {Function} becameValid
-  */
-  registerHandlers: function(target, becameInvalid, becameValid) {
-    this.on('becameInvalid', target, becameInvalid);
-    this.on('becameValid', target, becameValid);
-  },
```

Comment:
I don't know how many apps/addons rely on those events being triggered when registered via `model.get('errors').on('becameValid')` or `model.get('errors').registerHandlers()`. Since `Ember.Evented` is removed in this PR, code which relies on the `on` method being available on the `DS.Errors` throws an error.

The `registerHandler` method isn't advertised but it also hasn't been explicitly marked as `@private` so I technically this would break public API, I guess. If I am misinterpreting semver, please ignore me :pensive: .

Again, just to be clear: I am :+1: :+1: on removing those events and eventually make clearing the errors for an attribute an opt in solution, but I just want to point out possible implications for semver.

A possible, compatible solution could be to overwrite the `on` method and throw a deprecation...